### PR TITLE
rebuild chart (#443)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
And improve rebuild table

### CODE REVIEW

**Review by dinhlongviolin1 — APPROVED** [review]
Nicely done!

File: src/taipy/gui/utils/clientvarname.py [hunk 1]
```diff
@@ -9,11 +9,20 @@
 # an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
 # specific language governing permissions and limitations under the License.
 
+import typing as t
```
Comment (FabienLelaquais) [inline src/taipy/gui/utils/clientvarname.py]: Do you need that?

**Review by FabienLelaquais — APPROVED** [review]
Not pretending I get it all...
Looks really neat!

File: src/taipy/gui/utils/clientvarname.py (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/utils/clientvarname.py]: Good catch

**Pull request merged**